experimental/dds/tree2/package.json

```diff
@@ -33,8 +33,8 @@
 		"lint": "npm run prettier && npm run eslint",
 		"lint:fix": "npm run prettier:fix && npm run eslint:fix",
 		"postpack": "tar -cf ./experimental-tree2.test-files.tar ./src/test ./dist/test",
-		"prettier": "prettier --check . --ignore-path ../../../.prettierignore",
-		"prettier:fix": "prettier --write . --ignore-path ../../../.prettierignore",
+		"prettier": "prettier --check . --ignore-path .prettierignore",
+		"prettier:fix": "prettier --write . --ignore-path .prettierignore",
```

Comment:
I think we prefer to keep everything in the root `.prettierignore` due to the lack of inheritance and so changes there automatically apply everywhere... I think @tylerbutler might have more context or thoughts? I wouldn't object to adding the new entry to the root one.

Reply:
local prettierignores are completely unmanageable because there's no inheritance. The local ones _could_ be removed, but then when you run prettier manually from a folder (e.g. `pnpm exec prettier -w .` then it usually formats stuff it shouldn't. I don't know about other people, but I run prettier directly quite a bit because it's faster.

Anyway, IMO the prettier ignore story is near unusable in a mono repo. If it wasn't so slow, I would recommend always running `prettier -w .` even from the root and update the ignore files accordingly. But it's SO SLOW and has no caching.

Reply:
I guess I see both paths as having maintenance issues. The thing I like about the package-local approach is that it allows localizing the rules to what the package needs. Worst case, something is missed and an entry has to be added. I definitely prefer that to exploding the root config. Unless either of you feels particularly strongly in favor of the other approach, I'm going to mark this as resolved.

Reply:
I don't feel super strongly either way, but would err on the side of keeping to whatever we're doing now which seems to be the single root file, until we have a fully fledged story to change the approach.